frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemRow/uploadListItemRow.styles.ts

```diff
@@ -15,18 +15,16 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { DashboardListItemRow } from '@components/dashboard/dashboardList/dashboardListItem/components';
 import styled from 'styled-components';
 
-export const Container = styled.li`
+export const DashboardRow = styled(DashboardListItemRow)`
```

Comment:
Why is DashbordListItemRow being so heavily styled. Im probably missing something but for me it seems that the height is the only thing that is different.